joplin/api/schema.py

```diff
@@ -383,120 +478,335 @@ class Meta:
 
 
 class DepartmentPageTopPageNode(DjangoObjectType):
-    # This uses graphene ObjectType resolvers, see:
-    # https://docs.graphene-python.org/en/latest/types/objecttypes/#resolvers
-    service_page = graphene.Field(ServicePageNode)
-    information_page = graphene.Field(InformationPageNode)
-    guide_page = graphene.Field(GuidePageNode)
-    official_document_page = graphene.Field(OfficialDocumentPageNode)
+    title = graphene.String()
+    slug = graphene.String()
+    page_id = graphene.ID()
```

Comment:
instead of resolving the entire page for our top links, we just get the information we need to create the links